src/commonMain/kotlin/com/charleskorn/kaml/Annotations.kt

```diff
@@ -32,3 +32,38 @@ import kotlinx.serialization.SerialInfo
 public annotation class YamlComment(
     vararg val lines: String,
 )
+
+/**
+ * Write a String value if it is a single line in the specified ScalarStyle.
+ * This overrides the value specified in the [YamlConfiguration].
+ */
+@OptIn(ExperimentalSerializationApi::class)
+@Target(AnnotationTarget.PROPERTY)
+@Retention(AnnotationRetention.BINARY)
+@SerialInfo
+public annotation class YamlWriteSingleLineStringUsingScalarStyle(
```

Comment:
Perhaps `YamlSingleLineStringStyle` would be enough? This would also keep the symmetry with `YamlConfiguration`

(Similar comment applies for the multi-line annotation below)

Reply:
Yes, sounds good.